src/components/Interpretations/common/Message/Message.js

```diff
@@ -25,21 +25,19 @@ const Message = ({ children, text, created, id, username, onClick }) => (
                 display: flex;
                 flex-direction: column;
                 gap: ${spacers.dp8};
-                cursor: pointer;
             }
 
             .header {
                 display: flex;
                 gap: 6px;
                 align-items: center;
                 font-size: 13px;
-                line-height: 15px;
+                line-height: ${spacers.dp16};
```

Comment:
I think we've concluded that we should only use `spacers` for actual spacing (margins, padding, gaps etc). So in this case and in the `font-size: ${spacers.dp12};` case below, we should use regular values instead (`16px`, `12px` respectively).

Reply:
Fixed.